### PR TITLE
Component/display

### PR DESCRIPTION
**Pull request by nathanrodger**

### CODE REVIEW

File: cwel/component/display/display.gspec [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+= Spacers =
+
+    @on xs
+        display_wrap_*:
+            css margin-bottom is "56px"
+
+        display_title_*:
+            css margin-bottom is "14px"
+
+    @on s
```
Comment (dstuessy) [inline cwel/component/display/display.gspec]: @NathanRodger You should consider bundling the tests more often to save yourself some work:

```
@on s, m
    display_wrap_*:
        css margin-bottom is "64px"
    display_title_*:
        css margin-bottom is "16px"
```

File: cwel/component/display/display.gspec [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    display_title_*                            .display__title
+    display--center                            .display--center
+
+# http://galenframework.com/docs/reference-galen-spec-language-guide/#SpecsReference
+
+= Spacers =
+
+    @on xs
+        display_wrap_*:
+            css margin-bottom is "56px"
```
Comment (dstuessy) [inline cwel/component/display/display.gspec:15]: @NathanRodger I wouldn't test against margin-bottom as the design spec doesn't specify the use of margin or padding. Better would be to use `below`, `above`, `left-of`, or `right-of`: http://galenframework.com/docs/reference-galen-spec-language-guide/#BelowandAbove

File: cwel/component/display/display.gspec [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+    @on m
+        display_wrap_*:
+            css margin-bottom is "64px"
+
+        display_title_*:
+            css margin-bottom is "16px"
+
+    @on l
+        display_wrap_1:
```
Comment (dstuessy) [inline cwel/component/display/display.gspec]: @NathanRodger You can save yourself a lot of pain by using for loops: http://galenframework.com/docs/reference-galen-spec-language-guide/#forLoop

Seems like there's enough repetition to use loops.

File: cwel/component/display/display.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{% macro display(modifier, animation, image_modifier) %}
+<div class="display {[ modifier ]}">
+    <div class="display__container">
+
+        <div class="display__section">
+            <div class="display__image">
+                <div class="image {[ image_modifier ]}">
```
Comment (dstuessy) [inline cwel/component/display/display.html:7]: @NathanRodger Has the image component already got a design specification? I'm thinking this is a premature component implementation for something we haven't defined outside of the display component.

File: cwel/component/display/display.scss [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        @include grid-cell((
+            'md': 6
+        ));
+        @include breakpoint('md') {
+            padding-left: $s_md;
+        }
+    }
+    .display--reverse & {
+        @include breakpoint('md') {
+            @include grid-cell-gutter(md);
```
Comment (dstuessy) [inline cwel/component/display/display.scss]: @NathanRodger Does that render out successfully? just wondering if `md` should be in quotes.

File: cwel/component/display/display.scss [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+.display__title {
+    width: 100%;
+    margin-top: 0;
+    margin-bottom: $s_sm;
+}
+.display__text {
+    margin-bottom: $s_sm;
+}
+.display--top {
```
Comment (dstuessy) [inline cwel/component/display/display.scss]: @NathanRodger 🙈 what is this doing outside the `.display` ruleset? 😱

Comment (dstuessy) [reply cwel/component/display/display.scss]: @NathanRodger same with line 104!!!! 🙀

**Review by Decad — DISMISSED** [review]

File: cwel/style/variable/_vbl-colors.scss [hunk 7]
```diff
@@ -87,3 +87,9 @@ $c_icon_color_1: $c_primary;
 $c_icon_button_color_1: $white;
 $c_icon_button_color_2: $c_primary;
 
+// Image
+$c_image_bg: $c_bg-light;
+
+// Display
+$c_display_light_bg: $c_bg-light;
```
Comment (dstuessy) [inline cwel/style/variable/_vbl-colors.scss]: @NathanRodger I would keep variables for a component in the component's style file: `~/cwel/component/display/display.scss`, and add a `!default` flag so it's easier to theme.

**Review by dstuessy — CHANGES_REQUESTED** [review]

File: cwel/style/variable/_vbl-colors.scss (hunk 7 above)
Comment (nathanrodger) [reply cwel/style/variable/_vbl-colors.scss]: @dstuessy How are we meant to change the values for other themes if we do it this way?

File: cwel/component/display/display.html (hunk 4 above)
Comment (nathanrodger) [reply cwel/component/display/display.html:7]: @dstuessy Potentially yes however I beleive we dropped image as its own component. This was the first component that required an image so I added it following the spec that was outlined.

File: cwel/style/variable/_vbl-colors.scss (hunk 7 above)
Comment (dstuessy) [reply cwel/style/variable/_vbl-colors.scss]: `$c_display_light_bg: $c_bg_light !default;`

**Review by dstuessy — DISMISSED** [review]
Thanks, dude. Good stuff! 🙂

**Review by dstuessy — APPROVED** [review]
Good stuff. Merge it! :smile:

**Pull request merged**